index.html

```diff
@@ -30,14 +30,186 @@ <h1 class="info_box">
+                        <li><a href="#"><img src="/img/Count.svg" width="1" height="5" alt="count"></a></li>
+                        <li><a href="#"><img src="/img/bed.svg" height="22" alt="bed"></a></li>
+                        <li><a href="#"><img src="/img/cc.svg" width="1" height="5" alt="dot"></a></li>
+                        <li><a href="#"><img src="/img/time.svg" alt="date"></a></li>
+                    </ul>
+                </div>
+        </div>
+            <section>
+                <h3 class="p">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </h3>
+                <div>
+                    <ul class="push">
+                        <li class="html"><a href="#">html</a></li>
+                        <li class="css"><a href="#">CSS</a></li>
+                        <li class="javasc"><a href="#">Javascript</a></li>
+                    </ul>
+                </div>
+                <button class="endpush" type="button">See project</button> 
+            </section>
+    </section>
+
+
+    <section class="forth_work_container">
+        <div>
+            <img class="forth_wk" src="/img/wrkfour.jpg" alt="fourthwork">
+            <div class="workfour">
+                <h2>Multi-Post Stories</h2>
+            </div>
+                <div>
+                    <ul class="sec_h">
+                        <li><a href="#"><img src="/img/canopy.svg" alt="canopy"></a></li>
+                        <li><a href="#"><img src="/img/Count.svg" width="1" height="5" alt="count"></a></li>
+                        <li><a href="#"><img src="/img/bed.svg" height="22" alt="bed"></a></li>
+                        <li><a href="#"><img src="/img/cc.svg" width="1" height="5" alt="dot"></a></li>
+                        <li><a href="#"><img src="/img/time.svg" alt="date"></a></li>
+                    </ul>
+                </div>
+        </div>
+            <section>
+                <h3 class="p">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </h3>
+                <div>
+                    <ul class="push">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">CSS</a></li>
+                        <li><a href="#">Javascript</a></li>
+                    </ul>
+                </div>
+                <button class="endpush" type="button">See project</button> 
+            </section>
+    </section>
+
+
+    <section class="about_section_container">
+        <h2 class="about_my">About Myself</h2>
+        <h4 class="sec_abt">
+            Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </h4>
+        <div>
+            <p class="contact_section"><a href="#">LET'S CONNECT</a></p>
```

Comment:
Please stick to FIgma , here use the good color

<img width="166" alt="image" src="https://user-images.githubusercontent.com/7243132/170830581-582a37e4-ebb2-4ff3-831b-02b586656720.png">

Figma
<img width="166" alt="image" src="https://user-images.githubusercontent.com/7243132/170830607-1e058d21-3fc3-4a51-9e62-ef6ee6eb87b3.png">